streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/controller/ThreadingController.java

```diff
@@ -144,6 +167,33 @@ public Integer getWorkingNow() {
     }
 
     /**
+     * Lock the ThreadingController. If the threading controller has not
+     * yet been used the thread pool for this controller will be allocated
+     * at this time.
+     */
+    public void lock() {
+        synchronized (this) {
```

Comment:
Same comment as before, just folowing the pattern in the rest of the class / package. We can add a ticket to change back if we really care that much.